From: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Date: Thu, 2 Aug 2018 17:09:01 -0400
Subject: Generate old-style packets when possible

Old-style packets can support tag types from 0 to 15.  For packets
with higher tag numbers, we must use new-style packets.

Emitting packets this way makes the output more similar to GnuPG.
---
 package/tests/keyRing-test.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package/tests/keyRing-test.js b/package/tests/keyRing-test.js
index 1a5a3ce..d589d2c 100644
--- a/package/tests/keyRing-test.js
+++ b/package/tests/keyRing-test.js
@@ -295,7 +295,7 @@ test(withTestGpgHome(withEnigmail(function shouldImportFromTextAndGetKeyDetails(
   Assert.equal(pubKey.exitCode, 0);
 
   Assert.equal(pubKey.keyData.substr(0, 192),
-    "xsFNBFVHm5sBEACs94Ln+RMdeyBpWQtTZ/NZnwntsB10Wd3HTgo5sdA/OOFOJrWe" +
+    "mQINBFVHm5sBEACs94Ln+RMdeyBpWQtTZ/NZnwntsB10Wd3HTgo5sdA/OOFOJrWe" +
     "tJfAZ/HRxiSu1bwRaFVC8p061ftTbxf8bsdfsykYJQQqPODfcO0/oY2n/Z93ya8K" +
     "TzjXR3qBQ1P7f5x71yeuo7Zrj7B0G44Xjfy+1L0eka9paBqmm3U5cUew5wSr772L");
 
